### DISCUSSION TITLE
Assign PRs to CODEOWNERS users on self-hosted renovate with bitbucket server

### DISCUSSION THREAD

**Discussion by gomesp** [renovatebot/renovate#13227] [Request Help] [answered]
Hi team

I am running a self-hosted renovate bot with the following cofiguration:

```
module.exports = {
    platform: 'bitbucket-server',
    endpoint:  "https://my-company-bitbucket-server",
    username: process.env.USERNAME,
    password: process.env.RENOVATE_PASSWORD,
    logLevel: 'debug',
    onboardingConfig: {
      extends: ['config:base', ':semanticCommits', ':reviewersFromCodeOwners(true)', ':assigneesFromCodeOwners(true)'],
    },
    onboarding: false,
    printConfig: true,
    requireConfig: false,
    gitAuthor: "Pablo <pablo@my_company_name>",
    repositories: [ "~MY_USER/hello-world" ]
  }
```

The repository above has a ```CODEOWNERS``` file that is quite simple at the root:
```
/* USER1 USER2 USER3
```
For some reason the PRs won't get assigned by the users listed in the ```CODEOWNERS``` file. But if I add the following lines to the ```renovate.json``` file in the repository, then they will:

```
{
...
  "reviewersFromCodeOwners": true,
  "assigneesFromCodeOwners": true,
...
}
```

We are trying to avoid having any repository specific configuration. Is it possible to keep that config in the renovate ```config.js``` file only?

**Comment by gomesp** [comment, marked as answer]
Nevermind, I just realised that I needed to use the same syntax on the global config.js file.

**Reply by viceice** [reply]
`onboardingConfig` is only applied to onboarding PR, but `onboarding: false` prevents that. So you need to move the `extends` to toplevel config. 😉

**Reply by gomesp** [reply]
Great, thanks @viceice!
